components/applications-service/pkg/storage/postgres/schema/sql/11_add_denormalized_services_table.up.sql

```diff
@@ -1,3 +1,4 @@
+DROP TABLE IF EXISTS service_full;
```

Comment:
In migration 15, we drop the column `health_check_message`, but it is referenced in migration 12. Thus `CREATE TABLE IF NOT EXISTS` is not adequate, we need to drop the table.

Longer explanation
- (1st time) migration 11 runs, creates the table including the `health_check_message` column
- (1st time) migration 12 runs, the column is there
- (1st time) migration 15 removes the column
- (2nd time) migration 11 runs, doesn't create the table because it's already there, but its the one with `health_check_message` already removed
- (2nd time) migration 12 runs and tries to reference the `health_check_message` column which isn't there.

Dropping the table should be fine, in a production situation, the table should not exist or at worst we are immediately trying to recover after some intermittent failure halted this migration. In either case there shouldn't be data in the table.